confluence/client.py

```diff
@@ -463,6 +463,27 @@ def get_attachments(self, content_id, filename=None, media_type=None, expand=Non
                                        params=params,
                                        expand=expand)
 
+    def download_attachment(self, attachment):
+        # type: (Content) -> [byte]
+        """
+        Downloads an attachment. To be used in conjunction with get_attachments
```

Comment:
Please flag in the docstring here that this isn't a standard API route, slightly more likely that in the future when other types of auth are added someone would think to look here and note that we're using the self.client directly rather than _get method that's exposed.